packages/cli/src/config/beaconNodeOptions.ts

```diff
@@ -16,16 +16,13 @@ export class BeaconNodeOptions {
    */
   constructor({
     network,
-    configFile,
     beaconNodeOptionsCli,
   }: {
     network?: NetworkName;
-    configFile?: string;
     beaconNodeOptionsCli: RecursivePartial<IBeaconNodeOptions>;
   }) {
     this.beaconNodeOptions = mergeBeaconNodeOptions(
       network ? getNetworkBeaconNodeOptions(network) : {},
-      configFile ? readBeaconNodeOptionsIfExists(configFile) : {},
```

Comment:
Why not read from the config file? We want to *not* persist the options, but we do want to *read* them right? @wemeetagain

Reply:
ummm i thought after merging rcconfig we don't need to read options from any other config, i can change the name of the flag to something like `dumpConfig`. Let me know if I got the discussion wrong.